karmada-Failover-example/step3/text.md

```diff
@@ -0,0 +1,3 @@
+### Simulate taint
+
+RUN `karmadactl --kubeconfig /etc/karmada/karmada-apiserver.config taint cluster kind-member2 hello=:NoExecute`{{exec}}
```

Comment:
`failover-test=:NoExecute` is a better name?

Reply:
yeah , fixed it